testdata/identical_switch_conditions.go

```diff
@@ -0,0 +1,41 @@
+package fixtures
+
+func enforceSwitchStyle3() {
+
+	switch expression { // skipt tagged switch
+	case value:
+	default:
+	}
+
+	switch {
+	case a > 0, a < 0:
+	case a == 0:
+	case a < 0: // MATCH /case clause at line 11 has the same condition/
+	default:
+	}
```

Comment:
could you add a test about

```go
switch {
  case incr() > 0:
  case incr() > 0:
}
```

I'm unsure whether it should be reported with switch-identical-conditions. Or this pattern should be discouraged and this could be a candidate for another rule

Reply:
Recently I saw this:
```go
switch {
	case lnOpts.IsSocketOpts():
		// ...
		// check for timeout
		fallthrough
	case lnOpts.IsTimeout(), lnOpts.IsSocketOpts():
		// timeout listener with socket options.
		// ...
	case lnOpts.IsTimeout():
		// ...
	default:
		// ...
	}
```

Reply:
My point was to add a test about it.

Your example is good, it could be added